docs/tutorial/notebook/01_config_files.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<p>You can access configuration values as object attributes (<code>cfg.{property_name})</code> or dictionary key values (<code>cfg['{property_name}']</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Done</p>